lib/eex/test/eex/tokenizer_test.exs

```diff
@@ -13,148 +13,173 @@ defmodule EEx.TokenizerTest do
   end
 
   test "strings with embedded code" do
-    assert T.tokenize('foo <% bar %>', 1) ==
-           {:ok, [{:text, 'foo '}, {:expr, 1, '', ' bar '}]}
+    assert T.tokenize('foo <% bar %>', 1) == {:ok, [{:text, 'foo '}, {:expr, 1, '', ' bar '}]}
   end
 
   test "strings with embedded equals code" do
-    assert T.tokenize('foo <%= bar %>', 1) ==
-           {:ok, [{:text, 'foo '}, {:expr, 1, '=', ' bar '}]}
+    assert T.tokenize('foo <%= bar %>', 1) == {:ok, [{:text, 'foo '}, {:expr, 1, '=', ' bar '}]}
   end
 
   test "strings with embedded slash code" do
-    assert T.tokenize('foo <%/ bar %>', 1) ==
-           {:ok, [{:text, 'foo '}, {:expr, 1, '/', ' bar '}]}
+    assert T.tokenize('foo <%/ bar %>', 1) == {:ok, [{:text, 'foo '}, {:expr, 1, '/', ' bar '}]}
   end
 
   test "strings with embedded pipe code" do
-    assert T.tokenize('foo <%| bar %>', 1) ==
-           {:ok, [{:text, 'foo '}, {:expr, 1, '|', ' bar '}]}
+    assert T.tokenize('foo <%| bar %>', 1) == {:ok, [{:text, 'foo '}, {:expr, 1, '|', ' bar '}]}
   end
 
   test "strings with more than one line" do
-    assert T.tokenize('foo\n<%= bar %>', 1) ==
-           {:ok, [{:text, 'foo\n'}, {:expr, 2, '=', ' bar '}]}
+    assert T.tokenize('foo\n<%= bar %>', 1) == {:ok, [{:text, 'foo\n'}, {:expr, 2, '=', ' bar '}]}
   end
 
   test "strings with more than one line and expression with more than one line" do
     string = '''
-foo <%= bar
+    foo <%= bar
 
-baz %>
-<% foo %>
-'''
+    baz %>
+    <% foo %>
+    '''
 
-    assert T.tokenize(string, 1) == {:ok, [
+    exprs = [
       {:text, 'foo '},
       {:expr, 1, '=', ' bar\n\nbaz '},
       {:text, '\n'},
       {:expr, 4, '', ' foo '},
       {:text, '\n'}
-    ]}
+    ]
+
+    assert T.tokenize(string, 1) == {:ok, exprs}
   end
 
   test "quotation" do
-    assert T.tokenize('foo <%% true %>', 1) == {:ok, [
-      {:text, 'foo <% true %>'}
-    ]}
+    assert T.tokenize('foo <%% true %>', 1) == {:ok, [{:text, 'foo <% true %>'}]}
   end
 
   test "quotation with do/end" do
-    assert T.tokenize('foo <%% true do %>bar<%% end %>', 1) == {:ok, [
-      {:text, 'foo <% true do %>bar<% end %>'}
-    ]}
+    assert T.tokenize('foo <%% true do %>bar<%% end %>', 1) == {
+             :ok,
+             [{:text, 'foo <% true do %>bar<% end %>'}]
+           }
   end
 
   test "quotation with interpolation" do
-    assert T.tokenize('a <%% b <%= c %> <%= d %> e %> f', 1) == {:ok, [
+    exprs = [
       {:text, 'a <% b '},
       {:expr, 1, '=', ' c '},
       {:text, ' '},
       {:expr, 1, '=', ' d '},
       {:text, ' e %> f'}
-    ]}
+    ]
+
+    assert T.tokenize('a <%% b <%= c %> <%= d %> e %> f', 1) == {:ok, exprs}
+  end
 
-    assert T.tokenize('<%%% a <%%= b %> c %>', 1) == {:ok, [
+  test "improperly formatted quotation with interpolation" do
```

Comment:
This test originally belongs to "quotation with interpolation" on line 65 (there were originally two assertions in the test). I've moved it out and labeled it "improperly formatted quotation", which is what it is as far as I can tell.